aip/general/0127.md

```diff
@@ -54,8 +54,9 @@ message CreateBookRequest {
 
   // The user-specified ID for the book.
   // When using HTTP/JSON, this field is populated based on a query string
-  // argument, such as `?book_id=foo`. This is the fallback for fields that
+  // argument, such as `?bookId=foo`. This is the fallback for fields that
   // are not included in either the URI or the body.
+  // Note that clients convert the filed names to into camelCase format.
```

Comment:
How about: "Note that clients use camelCase format to communicate the id to the service." ?

I don't think the clients actually make the conversions.

Reply:
They actually do conversions (in our use case, the clients have proto messages that are snake_cased internally in many languages, and convert to camelCase), but it's an implementation detail we probably don't want to talk about.

I'm changing it to "Note that clients use camelCase format to communicate the field names to the service." (not just the "id" field, but all fields). Does this sound good?